src/MessageHandler/ZipDatasetFilesHandler.php

```diff
@@ -107,6 +117,9 @@ public function __invoke(ZipDatasetFiles $zipDatasetFiles)
             fclose($outputStream['fileStream']);
             $datasetSubmission->getFileset()->setZipFilePath($destinationPath);
             $this->entityManager->flush();
+
+            // Dispatch entity event.
+            $this->entityEventDispatcher->dispatch($datasetSubmission, 'dataset_zipped');
```

Comment:
I think we can remove this, and add it when we are going to listen for this event?

Reply:
Not every event always has a listener. But in our case we might remove it.